exercises/practice/grade-school/.meta/example.c

```diff
@@ -28,8 +28,8 @@ bool add_student(char *name, uint8_t grade)
 {
    bool added = false;
 
-   if (roster.count < MAX_STUDENTS) {
-      roster.students[roster.count].name = name;
+   if (roster.count < MAX_STUDENTS && strlen(name) <= MAX_NAME_LENGTH) {
```

Comment:
If we consider the string terminator to be part of the name length, then this can chane from `<=` to `<` and the `+1`'s won't be needed when declaring the `name[]`  string in the header file.

```suggestion
   if (roster.count < MAX_STUDENTS && strlen(name) < MAX_NAME_LENGTH) {
```

Reply:
Good suggestion, I made these changes.